terraform/environments/ops/main.tf

```diff
@@ -7,12 +7,18 @@ module "emblem_ops" {
   repo_name       = var.repo_name
 }
 
+data "google_cloud_run_service" "website_e2e_test_content_api" {
+  project  = var.app_project_id
+  name     = "content-api"
+  location = var.region
+}
+
 module "website_e2e_test" {
   source          = "../../modules/website-e2e-test"
-  count           = var.setup_e2e_tests ? 1 : 0
+  count           = var.setup_e2e_tests && var.setup_cd_system ? 1 : 0
```

Comment:
question: why 2 conditions?

Reply:
The `REPO_OWNER` and `REPO_NAME` variables are required by the E2E testing module:
- These variables are **not** provided to Terraform builds in `setup.sh` (where `setup_cd_system == False`)
  - This is the _first_ build in a typical `setup.sh` run
  - Initializing the E2E tests module in this build will cause Cloud Build to return a "repo not connected [or not found]" error
- They **are** provided to TF builds in the delivery-configuration script (where `setup_cd_system == True`)
  - This is the **second** build in a typical `setup.sh` run

In practice, this change prevents E2E tests from being set up if the CI/CD system is skipped.

Reply:
owner and name are provided via the default blank `""` in `variables.tf` and `setup_e2e_tests` is by default set to `false`.  `setup_cd_system` doesn't seem to have any effect.  

Double check that it will work using just a bool for `setup_e2e_tests` and let's go with just that.